editor/src/components/editor/action-types.ts

```diff
@@ -255,6 +255,7 @@ export type SetZIndex = {
 export type TransientActions = {
   action: 'TRANSIENT_ACTIONS'
   transientActions: Array<EditorAction>
+  elementsToRerender: Array<ElementPath> | null
```

Comment:
this could use the `export type ElementsToRerender = Array<ElementPath> | 'rerender-all-elements'` type here too for consistency

Reply:
that feels a bit like cheating, as the fixElements helper only works on filtering the result for transient actions